.github/workflows/deploy-github-pages.yaml

```
@@ -37,23 +37,25 @@ jobs:
 
     steps:
       - name: Install Hugo CLI
-        run: |
-          wget -O ${{ runner.temp }}/hugo.deb https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_extended_${HUGO_VERSION}_linux-amd64.deb \
-          && sudo dpkg -i ${{ runner.temp }}/hugo.deb
+        uses: peaceiris/actions-hugo@75d2e84710de30f6ff7268e08f310b60ef14033f #v3.0.0
+        with:
+          hugo-version: "latest"
+          extended: true
 
-      - name: Install Dart Sass
-        run: sudo snap install dart-sass
+      - uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 #v4.4.0
+        with:
+          node-version: "22.x"
```

Do we need to pin this version to `22.x`, can't we use `latest`?
```suggestion
          node-version: "latest"
```